doc/kinsol/guide/source/Usage/index.rst

```diff
@@ -1778,12 +1798,13 @@ The following optional outputs are available from the KINLS modules:
 User-supplied functions
 -----------------------
 
-The user-supplied functions consist of one function defining the nonlinear system,
-(optionally) a function that handles error and warning messages,
-(optionally) a function that handles informational messages,
-(optionally) one or two functions that provides Jacobian-related information for the linear
-solver, and (optionally) one or two functions that define the preconditioner for use in
-any of the Krylov iterative algorithms.
+The user-supplied functions consist of one function defining the nonlinear
+system, (optionally) a function that handles error and warning messages,
+(optionally) a function that handles informational messages, (optionally) one or
+two functions that provides Jacobian-related information for the linear solver,
+(optionally) one or two functions that define the preconditioner for use in
+any of the Krylov iterative algorithms, and (optionally) a function to compute
+the damping parameter in fixed-point or Picard iterations.
```

Comment:
Optional suggestion: convert to list since it's getting a bit long as a sentence